COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,201 @@
+pull requests to the io.js project.
+
+Collaborators should feel free to take full responsibility for
+managing issues and pull requests they feel qualified to handle, as
+long as this is done while being mindful of these guidelines, the
+opinions of other Collaborators and guidance of the TC.
+
+Collaborators may **close** any issue or pull request they believe is
+not relevant for the future of the io.js project. Where this is
+unclear, the issue should be left open for several days to allow for
+additional discussion. Where this does not yield input from io.js
+Collaborators or additional evidence that the issue has relevance, the
+issue may be closed. Remember that issues can always be re-opened if
+necessary.
+
+## Accepting Modifications
+
+All modifications to the the io.js code and documentation should be
+performed via GitHub pull requests, including modifications by
+Collaborators and TC members.
+
+All pull requests must be reviewed and accepted by a Collaborator with
+sufficient expertise who is able to take full responsibility for the
+change. In the case of pull requests proposed by an existing
+Collaborator, an additional Collaborator is required for sign-off.
+
+In some cases, it may be necessary to summon a qualified Collaborator
+to a pull request for review by @-mention.
+
+If you are unsure about the modification and are not prepared to take
+full responsibility for the change, defer to another Collaborator.
+
+Before landing pull requests, sufficient time should be left for input
+from other Collaborators. Leave at least 48 hours during the week and
+72 hours over weekends to account for international time differences
+and work schedules. Trivial changes (e.g. those which fix minor bugs
+or improve performance without affecting API or causing other
+wide-reaching impact) may be landed after a shorter delay.
+
+Where there is no disagreement (or discussion) amongst Collaborators,
+a pull request may be landed given appropriate review. Where there is
+discussion amongst Collaborators, consensus should be sought if
+possible. The lack of consensus may indicate the need to elevate
+discussion to the TC for resolution (see below).
+
+All bugfixes require a test case which demonstrates the defect. The
+test should *fail* before the change, and *pass* after the change.
+
+### Involving the TC
+
+Collaborators may opt to elevate pull requests or issues to the TC for
+discussion by assigning the ***tc-agenda*** tag. This should be done
+where a pull request:
+
+- has a significant impact on the codebase,
+- is inherently controversial; or
+- has failed to reach consensus amongst the Collaborators who are
+  actively participating in the discussion.
+
+The TC should serve as the final arbiter where required.
+
+## Landing Pull Requests
+
+Always modify the original commit message to include additional meta
+information regarding the change process:
+
+- A `Reviewed-By: Name <email>` line for yourself and any
```

Comment:
Are we any closer to scripting this? All of these metadata things are easy to fuck up and are begging to get scripted :)